CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Added
+[#364](https://github.com/plotly/dash-table/pull/224)
```

Comment:
364/224

Reply:
fixed in https://github.com/plotly/dash-table/pull/364/commits/12b1597590d76e88f551434e2b77bf51f236f120